app/components/footnote_modal_component/footnote_modal_component.html.erb

```diff
@@ -0,0 +1,21 @@
+<a href="#" data-toggle="modal" data-target="#<%= modal_id %>"><%= @title %>&nbsp;<span style="color: #007bff;"><%= icon %></span></a>
```

Comment:
This can't be part of the modal template, as the toggle could appear anywhere on the page, sometimes multiple times).